tests/conftest.py

```diff
@@ -148,6 +148,14 @@ def dev_root(request):
     return dev_root_path
 
 
+@pytest.fixture(scope='session')
+def install_openvino_dependencies(request):
+    if request.config.getoption('--product_version') < '2021.3':
+        return '/opt/intel/openvino/install_dependencies/install_openvino_dependencies.sh'
+    else:
+        return '/opt/intel/openvino/install_dependencies/install_openvino_dependencies.sh -y'
```

Comment:
add '-e' option as well for centos

Reply:
return '/opt/intel/openvino/install_dependencies/install_openvino_dependencies.sh -y -e'